### ISSUE TITLE
Recipe Submission Issue form

### ISSUE THREAD

**Issue by complexbrains** [neuroimaging-cookbook/neuroimaging-cookbook.github.io#26]
For this, we can simply follow the instructions [here[(https://www.youtube.com/watch?v=qQE1BUkf2-s&t=1s)

for which we will need .github folder for Issue and PR templated to be set in, for which can be created with a VScode extension [here](https://github.com/rhysd/dot-github) 

Or we can simply copy the folder from global2021 repo and start editing as we needed I believe it should work too?

First we will need to decide what text boxes we need, which of them will be required (obligatory) then the rest will be really easy to implement.

**Comment by shawnrhoads** [comment]
Awesome! We have a somewhat working form now (with #26 - #48). 

In the future, we will need to figure out how to work in the tags for software, modality, etc. Also, @dmoracze started working on a bot to create PRs from recipe sugestions (#28).

**Comment by shawnrhoads** [comment]
[Link to form](https://github.com/neuroimaging-cookbook/neuroimaging-cookbook.github.io/issues/new?assignees=shawnrhoads&labels=recipe&template=recipe-template.yml&title=%5BRecipe%5D%3A+)

**Comment by complexbrains** [comment]
Oh perfect! I kept checking both the project channel and DC channel but didn't see much activity there so thought maybe you postpone working on the project! Awesome to see it's going and loved the form! 

Let me know if there is anything else I can chime in, but seems all lovely as it is already🤗

**Comment by shawnrhoads** [comment]
Yeah, I think there is still a ton of room for improvement, but it's a start for the time being. Going to keep this issue open to discuss future improvements.
